src/storable.rs

```diff
@@ -584,3 +605,20 @@ pub(crate) const fn bytes_to_store_size(bounds: &Bounds) -> u32 {
         4
     }
 }
+
+#[cfg(test)]
+mod unit_tests {
```

Comment:
As for the tests, we added them in the same file because we did not know much about what to write as unit tests. Also, the module name "tests" conflicts with the module name "unit tests", so this module name was chosen to avoid that.

Reply:
The tests should be moved to the `src/storable/tests.rs` file, as that's where all the tests related to this code live.